src/herder/HerderImpl.cpp

```diff
@@ -949,8 +949,16 @@ HerderImpl::triggerNextLedger(optional<uint32_t> ledgerSeqToTrigger,
     // We pick as next close time the current time unless it's before the last
     // close time. We don't know how much time it will take to reach consensus
     // so this is the most appropriate value to use as closeTime.
+    auto const usingManualCloseWithoutExplicitCloseTime =
+        mApp.getConfig().MANUAL_CLOSE && !minCloseTime;
+    auto static const manualCloseDefaultPerLedgerTimeTIncrement =
```

Comment:
I'm not really understanding this constant. The `closeTime` has units of seconds, so you can just add 5. You don't need to convert it to a `time_t` or a `time_point` (which is not a duration).

Nitpick: We don't generally use local `static` variables, and I don't see any serious need for it here. Also can we change the name? It's a bit of an eye-sore and doesn't help me understand what it means.

Reply:
The code that implements the five-second default has been moved (to `ApplicationImpl::manualClose()`), and it no longer uses any conversions (the constant is just a `5`).